src/AccordionSection/AccordionSection.js

```diff
@@ -46,7 +46,9 @@ class AccordionSection extends Component {
       header,
       headerNote,
       panelId,
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       onHeaderClick,
```

Comment:
Linting pointed out a number of unused props.  I've punted by ignoring them for this changeset since this isn't a change in behavior and they can be revisited separately.

Reply:
interesting, since that prop *is* being used in the monolith